src/main/java/com/quantumsoft/qupathcloud/entities/DicomStores.java

```diff
@@ -18,15 +18,32 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * The DicomStores class contains a list of {@link com.quantumsoft.qupathcloud.entities.DicomStore}
+ * retrieving from the server.
+ *
```

Comment:
DicomStores contains a list of {@link com.quantumsoft.qupathcloud.entities.DicomStore}
 retrieved from the server.